Jack/Property.fs

```diff
@@ -221,28 +343,44 @@ module PropertyBuilder =
         member __.For(m : Property<'a>, k : 'a -> Property<'b>) : Property<'b> =
             Property.bind m k
 
+        member __.For(xs : seq<'a>, k : 'a -> Property<unit>) : Property<unit> =
+            let join m n =
+                Property.bind m (fun _ -> n)
+            let init =
+                Property.success ()
+            xs |> Seq.map k |> Seq.fold join init
```

Comment:
I've spent a bunch of time today on loops today, and I have something workable, but it can still blow the stack, we might need to build a trampoline in to Property/Gen/Random, not sure.